user/gdu/template.py

```diff
@@ -0,0 +1,18 @@
+pkgname = "gdu"
+pkgver = "5.29.0"
+pkgrel = 0
+build_style = "go"
+make_build_args = ["./cmd/gdu"]
+make_check_args = ["./cmd/gdu"]
```

Comment:
all the actual tests are in other folders too so leaving this as default is probably more correct

Reply:
(because this runs zero tests)